resources/migrations/000_migrations.yaml

```diff
@@ -11846,6 +11846,21 @@ databaseChangeLog:
             tableName: report_card
 
 
+  - changeSet:
+      id: v44.00-029
+      author: qnkhuat
+      comment: Added 0.44.0 - Add exceeded_limit to metabase_field
+      changes:
+        - addColumn:
+            tableName: metabase_field
+            columns:
+              - column:
+                  name: exceeded_limit
+                  type: boolean
+                  remarks: Whether or not a field has values store in metabase_fieldvalues is exceeded the limits
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

Comment:
It seems weird to add a `exceeded_limit` column to Field to record whether or not its FieldValues have exceeded some limit... why not add this column to FieldValues?

Reply:
Failing that something like `field_values_exceeded_limit` would be a more appropriate name. If I see a Field with `{:exceeded_limit false}` I have no idea what that means unless I go read this migration here